src/java_bytecode/java_string_library_preprocess.h

```diff
@@ -52,7 +52,9 @@ class java_string_library_preprocesst:public messaget
   void add_string_type(const irep_idt &class_name, symbol_tablet &symbol_table);
   bool is_known_string_type(irep_idt class_name);
 
+#ifndef UNIT_TEST
```

Comment:
The particular macro name seems likely to clash with macros used within a unit testing framework, suggest a CBMC_ or similar prefix.